### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1002] [Galaxy Trivia] [answered]
```
e|---------------------------------------|
B|--------0--1---------------------------|
G|-----------------------------0-2-------|
D|-----2--------2-0h2p0-------------3--0-|
A|--0--------------------3---3-----------|
E|-------------------------1-------------| (x4)
```

**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment]
The song is **Californication** by American rock band Red Hot Chili Peppers.

The song Californication was **written by the members of the band Red Hot Chili Peppers: Anthony Kiedis (lead vocalist), Flea (bassist), John Frusciante (guitarist), and Chad Smith (drummer)**. They collectively composed the song, which was released as the title track of their 1999 album "Californication." **The lyrics were primarily penned by Anthony Kiedis**, with musical contributions from the other band members.

**Comment by thuongtruong1009** [comment, marked as answer]
The provided guitar tab appears to be a portion of the song "Wonderwall" by Oasis. The songwriter of "Wonderwall" is Noel Gallagher, who is a member of the band Oasis.
